clients/go/pkg/zbc/client.go

```diff
@@ -36,7 +36,6 @@ import (
 	"github.com/camunda-cloud/zeebe/clients/go/pkg/worker"
 )
 
-const DefaultRequestTimeout = 15 * time.Second
```

Comment:
:wrench: We could keep this around and use it here: https://github.com/camunda-cloud/zeebe/blob/4bd886dbabb32741ba0a2cca48f1e1ac323ed0b1/clients/go/cmd/zbctl/internal/commands/root.go#L92

This would prevent the (technically speaking) breaking change and also match how other cli parmaeter defaults are set.

Reply:
It's how it used to be, but I personally find it confusing. Why do we have a default value in the client, which is only used in `zbctl`? So effectively, it's the `zbctl` default request timeout, no? That was my reasoning for the change. I also figured it might be confusing to users, who might expect it to be the actual default request timeout, when it's not true. If a user doesn't pass a `context.WIthTimeout` or `context.WithDeadline` when sending a request, then the default gRPC timeout is used.